integration_test/tests/wallet.rs

```diff
@@ -279,6 +279,30 @@ fn wallet__get_transaction__modelled() {
     model.unwrap();
 }
 
+#[test]
+fn wallet__import_address() {
+    let node = match () {
+        #[cfg(feature = "v22_and_below")]
+        () => Node::with_wallet(Wallet::Default, &[]),
+        #[cfg(not(feature = "v22_and_below"))]
+        () => {
+            let node = Node::with_wallet(Wallet::None, &["-deprecatedrpc=create_bdb"]);
```

Comment:
Just to comment that this feature gating is required since the RPC is only supported on legacy wallets. Opposed to `encryptwallet` in the other recent PR.

This method of creating a legacy wallet works up to v29 and the other tests like `dumpwallet` and `dumpprivkey` could be updated to match this format.